clang/lib/Format/ContinuationIndenter.cpp

```diff
@@ -593,7 +593,8 @@ bool ContinuationIndenter::mustBreak(const LineState &State) {
         // name.
         !Style.isJavaScript()) ||
        (Current.is(tok::kw_operator) && Previous.isNot(tok::coloncolon))) &&
-      Previous.isNot(tok::kw_template) && CurrentState.BreakBeforeParameter) {
+      Previous.isNot(tok::kw_template) && CurrentState.BreakBeforeParameter &&
+      (Style.isCpp() && Current.Tok.isNot(tok::kw_operator))) {
```

Comment:
Two things, you don't need the parenthesis, you are already in a conjunction.
Does it affect the other languages, that you now add a check for `isCpp()`?

Reply:
I am not sure if affects other languages, so put it there. I will remove it

Reply:
Fixed